core/assocs/assocs-docs.factor

```diff
@@ -328,6 +328,8 @@ HELP: assoc-partition
 }
 { $description "Calls a predicate quotation on each key of the input assoc. If the test yields true, the key/value pair is added to " { $snippet "true-assoc" } "; if false, it's added to " { $snippet "false-assoc" } "." } ;
 
+{ assoc-partition partition } related-words
```

Comment:
I'm not sure we should make all the ``assoc-foo`` and ``foo`` words related.  Long term I think we should have a common cursors/containers interface so operating on hashtables and arrays isn't so different.  But maybe this isn't a big deal. Dunno.